src/api/spec/browser_helper.rb

```diff
@@ -7,3 +7,7 @@
 
 # helper methods for authentification in features tests
 require 'support/features/features_authentification'
+
+# Shared examples. Per recommendation of RSpec,
+# https://www.relishapp.com/rspec/rspec-core/v/2-12/docs/example-groups/shared-examples
+Dir["./spec/support/shared_examples/features/*.rb"].each { |example| require example }
```

Comment:
@openSUSE/open-build-service In this PR I needed to use the shared examples among different test files. RSpec maintainers recommended for this to store such examples in spec/support/*. Since we have different test environmentes, like feature and controller tests, I assumed that it might be a good idea to split those examples likewhise.

Please speak up if you are not ok with this approach:)